### ISSUE TITLE
Repository link not updated in README

### ISSUE THREAD

**Issue by lucarei** [vincentmllr/deep-robot-grasping#1]
Hi. Thanks for your works. I tried to clone the panda simulation but username and password are needed. Can u help me?

**Comment by TheQuyen97ht** [comment]
I got the same problem as yours. Could anyone provide us the account information ?

**Comment by vincentmllr** [comment]
Hi, sorry for the late reply. I will have a look if I can help you. Just to clarify: Could you give me some more details on which account details you need. Maybe you can tell me at which step in the README you are?

**Comment by lucarei** [comment]
Hi, I have tested your repo some time ago but I remember that I was not able to clone this link “ git clone https://git.scc.kit.edu/uhksn/panda-deep-grasping.git ” since username and password are needed. I am still interested now in your project and if you can share it I would greatly appreciate it.

**Comment by vincentmllr** [comment]
Thank you for your message! Now I see what the problem is: I transferred this repo from my universities gitlab to my private github account and didnt update mentioned command in the README. The correct command should be: `git clone https://github.com/vincentmllr/deep-robot-grasping`.
I am not sure if I set the authorizations right so you can clone. I will check that too.
I will also update the README soon. Please let me know if that solved the issue.

**Comment by flydragon2018** [comment]
different repo name,  same repo?

**Comment by vincentmllr** [comment]
> different repo name,  same repo?

Yes, it was called `panda-deep-grasping` in my universities account and is called different now after moving it to my personal account.
